src/test/java/uk/gov/dhsc/htbhf/steps/CommonSteps.java

```diff
@@ -1,27 +1,109 @@
 package uk.gov.dhsc.htbhf.steps;
 
-import io.cucumber.java.en.Given;
-import io.cucumber.java.en.When;
-import uk.gov.dhsc.htbhf.page.BasePage;
-import uk.gov.dhsc.htbhf.page.GuidancePage;
-import uk.gov.dhsc.htbhf.page.PageName;
+import uk.gov.dhsc.htbhf.page.*;
```

Comment:
Just an observation - this class could get rather large

Reply:
True, the same goes for common steps in the JavaScript version as they are mirrors of each other. We can keep our eye on it and tidy up/restructure if it becomes a problem.